packages/clay-core/src/vertical-nav/VerticalNav.tsx

```diff
@@ -49,6 +49,11 @@ type Props<T extends Record<string, any> | string> = {
 	 */
 	activation?: 'manual' | 'automatic';
 
+	/**
+	 * Flag to define aria-label.
+	 */
+	ariaLabel: string;
```

Comment:
If you want to keep the same pattern as React, we can also keep the same `aria-label` naming here. I've been trying to do this to maintain the standard. You can take a look at the picker.

Reply:
Done!